Source/ACE/Entity/Creature.cs

```diff
@@ -115,6 +128,54 @@ private void SetAbilities(AceCreatureObject aco)
             abilities.Add(Enum.Ability.Mana, Mana);
 
             Abilities = new ReadOnlyDictionary<Enum.Ability, CreatureAbility>(abilities);
+
+            IsAlive = true;
+        }
+
+        public void Kill(Session session)
+        {
+            IsAlive = false;
+
+            // Send UpdateHealth Message with 0 Health - this is not necessary according to live pcaps
+            // var healthLossEvent = new GameEventUpdateHealth(session, Guid.Full, 0f);
+            // session.Network.EnqueueSend(healthLossEvent);
+
+            // 1. step in retail: Create and send the death notice
+            string killMessage = $"{session.Player.Name} has killed {Name}.";
+            var creatureDeathEvent = new GameEventDeathNotice(session, killMessage);
+            session.Network.EnqueueSend(creatureDeathEvent);
+
+            // 2. step in retail: SoundEvent: sound 48 (HitFlesh1), volume 0,5
+            session.Player.ActionApplySoundEffect(Sound.HitFlesh1, Guid);
+
+            // 3. step in retail: SoundEvent: sound 14 (Wound3), volume 1
+            session.Player.ActionApplySoundEffect(Sound.Wound3, Guid);
+
+            // 4. step in retail: MovementEvent: (Hand-)Combat to Death
+            // TODO: this isn't really working yet, we need more work on the motion stuff
+            GeneralMotion motion1 = new GeneralMotion(MotionStance.Standing, new MotionItem(MotionCommand.Dead));
+            session.Network.EnqueueSend(new GameMessageUpdateMotion(this, session, motion1));
```

Comment:
I expect this method to change again after some more refactoring but for now...

change `session.Network.EnqueueSend(new GameMessageUpdateMotion(this, session, motion1));` to `session.Player.EnqueueMovementEvent(motionDeath, this.Guid);`